development-defaults.ini

```diff
@@ -226,68 +185,4 @@ deadline = "2014-10-15"
-name = "Approve/Decline Requests to Help With Setup/Teardown"
-deadline = "2014-12-11"
-description = "An overwhelming majority of staffers want to work setup and teardown shifts rather than work during the event itself, so we have far more offers than we have need for.  Since this affects what hotel nights staffers get, please approve and decline requests for this for people in your department."
-path = "/hotel/requests?department={department}"
-
-[[hotel_assignments]]
-name = "Assign Hotel Roomates You KNOW Want To Room Together"
-deadline = "2014-12-11"
-description = "STOPS manages hotel rooms, and we're happy to make hotel room assignments for the staffers for all of the departments. However, some department heads prefer to manage hotel room assignments for some or all of their staff. DO NOT ASSIGN STAFFERS TO ROOMS RANDOMLY! ONLY CREATE ROOMS AND ASSIGN PEOPLE THERE IF YOU KNOW THOSE PEOPLE WANT TO ROOM TOGETHER!"
-path = "/hotel/assignments?department={department}"
-
-[[placeholders]]
-name = "Checking Placeholder Registrations"
-deadline = "2014-12-15"
-description = "We create placeholder registrations for volunteers and ask them to fill out the rest of their information and also confirm that they'll be coming.  We need our department heads to review the unclaimed badges for their departments to check for any essential volunteers who haven't claimed their badges."
-path = "/registration/placeholders?department={department}"
-
-[[postcon_hours]]
-name = "(After the Event) Marking and Rating Shifts"
-description = "After the weekend is over, we'll want all department heads to ensure that their volunteers had their shifts marked and rated."
-deadline = "2015-02-02"
-path = "/jobs/signups?location=59983785"
-
-[[tech_requirements]]
-deadline = "2014-12-07"
-description = "What do you need in terms of laptops, projectors, cables, internet access, etc?"
-
-[[printed_signs]]
-deadline = "2014-12-07"
-description = "Other than a sign for your area, what printed signs/banners/forms do you need?"
-
-[[office_supplies]]
-deadline = "2014-12-07"
-description = "Do you need any paper, pens, sharpies, tripods, whiteboards, scissors, staplers, etc?"
-
-[[custom_ribbons]]
-deadline = "2014-12-07"
-description = "Besides the ribbons given out at registration, you may need your own ribbon type to help manage access to certain restricted areas. If so, please let us know what they should say, how many you'll need, and what color you'd like them to be. These ribbons will be given to you at the beginning of MAGFest to distribute to whoever needs them.  (Note: Small orders of ribbons are very expensive - please request a ribbon type only if you need it, and tell us why you need it so we can make sure there's no duplication between departments.)"
-
-[[treasury]]
-name = "Cash / MPoint Needs"
-deadline = "2014-12-19"
-description = "Tell us whether you need any cash/mpoints for your department."
-path = "/dept_checklist/treasury"
+[dept_head_checklist]
```

Comment:
It doesn't hurt to have this here, but there's no need to have an empty config section here.  We could just omit it entirely and it would be the same thing.